syntaxes/Dafny.tmLanguage

```diff
@@ -16,7 +16,7 @@
 	<array>
 		<dict>
 			<key>begin</key>
-			<string>(class)\s*([\w'?]+\b(?!&lt;))?\s*</string>
+			<string>(class\b)\s*([\w'?]+\b(?!&lt;))?\s*</string>
```

Comment:
```suggestion
			<string>(class)\s+([\w'?]+\b(?!&lt;))?\s*</string>
```

Reply:
Your solution is fine too, and I'm not sure whether `\s` allows a newline, but still putting this here.

Reply:
I see what you mean, but that would prevent the parsing of `class{:extern}name` properly.